testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -22,15 +22,30 @@
     SUPPORTS_PREDICATE_PUSHDOWN,
     SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY(SUPPORTS_PREDICATE_PUSHDOWN),
     SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY(SUPPORTS_PREDICATE_PUSHDOWN),
+
     SUPPORTS_LIMIT_PUSHDOWN,
+
     SUPPORTS_TOPN_PUSHDOWN,
+
     SUPPORTS_AGGREGATION_PUSHDOWN,
+
     SUPPORTS_JOIN_PUSHDOWN(
             // Currently no connector supports Join pushdown by default. JDBC connectors may support Join pushdown and BaseJdbcConnectorTest
             // verifies truthfulness of SUPPORTS_JOIN_PUSHDOWN declaration, so it is a safe default.
             false),
     SUPPORTS_JOIN_PUSHDOWN_WITH_FULL_JOIN(SUPPORTS_JOIN_PUSHDOWN),
     SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM(SUPPORTS_JOIN_PUSHDOWN),
+
+    SUPPORTS_CREATE_TABLE,
+    SUPPORTS_CREATE_TABLE_WITH_DATA(SUPPORTS_CREATE_TABLE),
+
+    SUPPORTS_INSERT,
+
+    SUPPORTS_DELETE(false),
+    SUPPORTS_ROW_LEVEL_DELETE(SUPPORTS_DELETE),
```

Comment:
I don't get. Feature `SUPPORTS_ROW_LEVEL_DELETE` is by default set to `SUPPORTS_DELETE` which is by default disabled. How should I read this?

Reply:
You should read this as:

- if connector declares that it supports row level delete, then it should be expected to work
- if connector declares it supports delete (and doesn't say anything about row level delete), then it should be expected to work
- if connector declares it supports delete, but declares no support for row level delete, then it should be expected _not_ to work
- if connector declares nothing, _no_ delete should be expected to work

Reply:
> if connector declares it supports delete, but declares no support for row level delete, then it should be expected not to work

You mean it supports delete and does not support row level delete. Does row level delete indicates delete?

Did you mean delete -> metadata delete and row level delete -> delete?

Reply:
> Does row level delete indicates delete?

yes, if a connector supports row level delete, then it does support delete in general